AlembicImporter/Assets/UnitTests/AlembicExportTest.cs

```diff
@@ -126,110 +116,268 @@ public virtual void Term()
-                Directory.Delete(filePath, recursive: true);
-                AssetDatabase.Refresh();
-            }
-            catch (IOException)
-            {
+            try {
+                Directory.Delete (filePath, recursive : true);
+                AssetDatabase.Refresh ();
+            } catch (IOException) {
                 // ignore -- something else must have deleted this.
             }
         }
+
     }
 
-    public class AlembicExportTest : AlembicTestBase
-    {
+    public class AlembicExportTest : AlembicTestBase {
         private const string c_scene = "TestCloth";
-
-        private AlembicExporter GetAlembicExporter()
-        {
-            var alembicExporter = Object.FindObjectOfType<AlembicExporter>();
-            Assert.That(alembicExporter, Is.Not.Null);
+        private const string cd_scene = "TestCreateAndDelete";
+        private const string cc_scene = "TestCustomCapturer";
+        private const string ce_scene = "TestExport";
+        private const string cgui_scene = "TestGUI";
+        private AlembicExporter selectedExporter;
+        private string selectedExportPath;
+
+        private AlembicExporter GetAlembicExporter () {
+            var alembicExporter = Object.FindObjectOfType<AlembicExporter> ();
+            Assert.That (alembicExporter, Is.Not.Null);
             return alembicExporter;
         }
 
-        private string GetAssetsAbsolutePath(string relPath)
-        {
-            return Application.dataPath + relPath.Replace("Assets", "");
+        private string GetAssetsAbsolutePath (string relPath) {
+            return Application.dataPath + relPath.Replace ("Assets", "");
         }
 
         /// <summary>
         /// Test that the abc file was reimported and can be loaded into Unity through
         /// the asset database.
         /// </summary>
         /// <param name="abcPath"></param>
-        private void TestAbcImported(string abcPath)
-        {
-            AssetDatabase.Refresh();
+        private void TestAbcImported (string abcPath) {
+            AssetDatabase.Refresh ();
 
-            var absPath = GetAssetsAbsolutePath(abcPath);
+            var absPath = GetAssetsAbsolutePath (abcPath);
 
-            Assert.That(string.IsNullOrEmpty(absPath), Is.False);
-            Assert.That(File.Exists(absPath));
+            Assert.That (string.IsNullOrEmpty (absPath), Is.False);
+            Assert.That (File.Exists (absPath));
 
             // now try loading the asset to see if it imported properly
-            var obj = AssetDatabase.LoadMainAssetAtPath(abcPath);
-            Assert.That(obj, Is.Not.Null);
+            var obj = AssetDatabase.LoadMainAssetAtPath (abcPath);
+            Assert.That (obj, Is.Not.Null);
             var go = obj as GameObject;
-            Assert.That(go, Is.Not.Null);
+            Assert.That (go, Is.Not.Null);
+        }
+
+        // Loads a given scene, generates a random export path and sends out the alembic exporter
```

Comment:
this comment seems to be saying the function does a bit more than it actually does.